wiki/forms.py

```diff
@@ -195,6 +200,9 @@ def clean(self):
             raise forms.ValidationError(
                 ugettext(
                     'While you were editing, someone else changed the revision. Your contents have been automatically merged with the new contents. Please review the text below.'))
+        if ('title' not in cd) or (not cd['title'].strip()):
```

Comment:
I would suggest adding a `clean_title` method instead since that will prompt the error message to show up next to the title field.

```python
def clean_title(self):
    title = self.cleaned_data.get('title')
    title = (title or "").strip()
    if not title:
        raise forms.ValidationError(ugettext('Article is missing title or has an invalid title'))
    return title